Sprint-1/errors/3.js

```diff
@@ -1,5 +1,5 @@
 const cardNumber = 4533787178994213;
-const last4Digits = cardNumber.slice(-4);
+const last4Digits = Number(cardNumber.toString().slice(-4));
```

Comment:
Is it practical to convert the last 4 digits back to Number type? What if the last 4 digits is 0000 
(No change needed, just a question to think about)

Reply:
That's a good point. It's impractical to convert it to a number because if I convert '0000' to a number, it becomes 0, which isn't what I intended.